Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -234,7 +234,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
                   "counterclockwise otherwise."),
                 _("Rotate _PARAM0_ at speed _PARAM1_ deg/second"),
                 _("Angle"),
-                "res/actions/direction24.png",
+                "res/actions/rotate24_black.png",
```

Comment:
I see this is not the same icon, but I assume this was a mistake before?

Reply:
I decided that actions that mentionne "Rotate" should have this icon.